frontend/src/app/images/images.component.html

```diff
@@ -0,0 +1,21 @@
+<div class="clr-row clr-justify-content-center">
+  <div class="clr-col-4">
+    <h1>Is this image a boy or a girl?</h1>
+    <h2 [hidden]="clickMessage == null">{{ clickMessage }}</h2>
```

Comment:
Personally, I try to use a 'strictly equals' check in JS/TS. One of my mentors in the past suggested this to me early on, and I've been following it ever since. I've _really_ only been burned by it a handful of times, but I can say that definitely since i switched i've not had any issues. 

Another note, for super-extra-mega bonus points, we can use getters in the component logic to clean up our templates. My general rule is, "If there's more than one or two conditional checks, outsource it to a getter".

Here's a quick article on the topic: https://codeburst.io/javascript-double-equals-vs-triple-equals-61d4ce5a121a